test/unit/components/shepherd-button.spec.js

```diff
@@ -93,6 +93,21 @@ describe('component/ShepherdButton', () => {
       expect(button).toHaveAttribute('aria-label', '5');
     });
 
+    it('label - funtion', () => {
+      const config = {
+        label: () => 'Test'
```

Comment:
@radibit should this test do something to change the value returned by the function after some time and check the value was updated? I assume that is the dynamic updating you are trying to achieve?

Reply:
@rwwagner90 By dynamic, I've meant making the `label` and `text` values as a result of a function call next to the option to be primitives of type `string`. That is also what the unit tests I've added are representing.

The use case I had (and successfully tested with the version of this PR) is an i18n solution which has to re-render the content of the app as a callback for changing the language settings, including the tour steps managed by Shepherd. To achieve that, I had to make sure that all text values are functions connected to that i18n solution which will return me the respected translation.

Reply:
@radibit I understand your use case, but I am not sure I understand if these are supposed to update or not. I thought, what we were doing here is using the function instead of a string, so that every time the step was shown, the function would be called, and the text would dynamically change. If this is the case, we should update the tests to have a function that changes what it returns, so we can test that when the function returns something different the step shows it. Does that make sense?

Reply:
@rwwagner90 Thank you for the clarification. I've got your point and updated the `button` unit tests for the dynamic `text` and `label` properties.

Ref: [9ca563c](https://github.com/shipshapecode/shepherd/pull/1815/commits/9ca563c72e70cf28495ef96546d8bfd45ea2de31)